spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,162 @@
+require 'rails_helper'
```

Comment:
incorrect file location
namespaced controllers should be located in sudirectory for spec/controllers as they are in app/controllers

Reply:
move it to account folder?